CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
 Add:  POST /v2/entities - entity creation under version 2 (Issue #970)
 Add:  API version 2 only supports JSON so we can now distinguish between STRINGS, NUMBERS (all treated as floats for now),
       and BOOLEANS (true, false), for the VALUE for ContextAttribute and Metadata (No Issue)
+Fix:  Fixed a bug about error handling for API version 2 (Issue #1087)
```

Comment:
Any error with 'invalid tenant', 'invalid service path', etc   would produce this 'ex bug'